sci-electronics/kicad/kicad-4.0.0_rc1.ebuild

```diff
@@ -0,0 +1,183 @@
+		# add the libraries directory subproject also to uninstallation procedures
+		sed '/make uninstall/,/# /d' -i ${PN}-library/CMakeLists.txt || die "sed failed"
+	fi
+
+	# Add internationalization for the GUI
+	if use i18n; then
+		# create a link to the translations library in the main project folder
+		ln -s "${WORKDIR}/${P}-i18n" "${S}/${PN}-i18n" || die "ln failed"
+		# Remove unused languages. Project generates only languages specified in the
+		# file in LINGUAS in the subproject folder. By default all languages are added
+		# so we sed out the unused ones based on the user linguas_* settings.
+		local lang=""
+		for lang in ${LANGS}; do
+			if ! use linguas_${lang}; then
+				sed "/${lang}/d" -i ${PN}-i18n/LINGUAS || die "sed failed"
+			fi
+		done
+		# cmakelists does not respect our build dir variables, so make it point to the right location
+		sed "s|\${CMAKE_BINARY_DIR}|${WORKDIR}/${P}_build|g" -i ${PN}-i18n/CMakeLists.txt || die "sed failed"
+		# we also make from the master project so the source dir is understood incorretly, replace that too 
+		sed "s|\${CMAKE_SOURCE_DIR}/\${LANG}|\${CMAKE_SOURCE_DIR}/${PN}-i18n/\${LANG}|g" -i ${PN}-i18n/CMakeLists.txt || die "sed failed"
+		# add the translations directory to cmake as a subproject to build
+		sed "/add_subdirectory( bitmaps_png )/a add_subdirectory( ${PN}-i18n )" -i CMakeLists.txt || die "sed failed"
+		# add the translations directory subproject also to uninstallation procedures
+		sed '/make uninstall/,$d' -i ${PN}-i18n/CMakeLists.txt || die "sed failed"
+	fi
+
+	# Install examples in the right place if requested
+	if use examples; then
+		sed -e 's:${KICAD_DATA}/demos:${KICAD_DOCS}/examples:' -i CMakeLists.txt || die "sed failed"
+	else
+		sed -e '/add_subdirectory( demos )/d' -i CMakeLists.txt || die "sed failed"
+	fi
+
+	# Add important missing doc files
+	sed -e 's/INSTALL.txt/AUTHORS.txt CHANGELOG.txt README.txt TODO.txt/' -i CMakeLists.txt || die "sed failed"
+}
+
+src_configure() {
+	need-wxwidgets unicode
+
+	local mycmakeargs=(
+		-DKICAD_DOCS="/usr/share/doc/${PF}"
+		-DKICAD_HELP="/usr/share/doc/${PF}/help"
+		-DwxUSE_UNICODE=ON
+		-DKICAD_SKIP_BOOST=ON
+		$(cmake-utils_use github BUILD_GITHUB_PLUGIN)
+		$(cmake-utils_use python KICAD_SCRIPTING)
+		$(cmake-utils_use python KICAD_SCRIPTING_MODULES)
+		$(cmake-utils_use python KICAD_SCRIPTING_WXPYTHON)
+		$(cmake-utils_use webkit KICAD_USE_WEBKIT)
+		$(usex i18n "-DKICAD_I18N_UNIX_STRICT_PATH=1" "")
+	)
+	if use debug; then
+		append-flags "-DDEBUG"
+	fi
+	cmake-utils_src_configure
+}
+
+src_compile() {
+	cmake-utils_src_compile
+	if use doc; then
+		doxygen Doxyfile || die "doxygen failed"
+	fi
+}
+
+src_install() {
+	cmake-utils_src_install
+	use python && python_optimize "${D}/usr/$(get_libdir)/${EPYTHON}/site-packages/pcbnew.py"
+	if use doc ; then
+		insinto /usr/share/doc/${PF}
+		doins uncrustify.cfg
```

Comment:
dodoc uncrustify.cfg

We used `doins` to install files without compression in the past but this is no longer the case in EAPI 5.